Marlin/SanityCheck.h

```diff
@@ -331,9 +331,9 @@
 /**
  * Only one type of extruder allowed
  */
-#if (ENABLED(SWITCHING_EXTRUDER) && (ENABLED(SINGLENOZZLE) || ENABLED(MIXING_EXTRUDER))) \
+#if (ENABLED(SWITCHING_EXTRUDER) &&  ENABLED(MIXING_EXTRUDER)) \
   || (ENABLED(SINGLENOZZLE) && ENABLED(MIXING_EXTRUDER))
-    #error "Please define only one type of extruder: SINGLENOZZLE, SWITCHING_EXTRUDER, or MIXING_EXTRUDER."
+    #error "Please define only one type of extruder: SINGLENOZZLE/SWITCHING_EXTRUDER or MIXING_EXTRUDER."
```

Comment:
The condition can be simplified.
```cpp
#if ENABLED(MIXING_EXTRUDER) && (ENABLED(SWITCHING_EXTRUDER) || ENABLED(SINGLENOZZLE))
```
But maybe we should break this into 2 simple errors:
```cpp
#if ENABLED(MIXING_EXTRUDER)
  #if ENABLED(SWITCHING_EXTRUDER)
    #error "MIXING_EXTRUDER is not compatible with SWITCHING_EXTRUDER."
  #elif ENABLED(SINGLENOZZLE)
    #error "MIXING_EXTRUDER is not compatible with SINGLENOZZLE."
  #endif
```